HipoProject/urls.py

```diff
@@ -21,7 +21,9 @@
 #                            DeleteRecipeView
 #                            )
 from django.urls import path
-from recipes.views import ListCreateIngredientView, ListCreateRecipeView, RecipeDetailView, ListCreateDeleteLikesView
+from recipes.views import (ListCreateIngredientView, ListCreateRecipeView, RecipeDetailView,
+                           ListCreateDeleteLikesView, CreateUpdateRatesView
+                           )
```

Comment:
You do not have to use parenthesis, you can import like this
`from recipes.views import ListCreateIngredientView, ListCreateRecipeView, RecipeDetailView, \
    ListCreateDeleteLikesView, CreateUpdateRatesView` 

For your interest.